### PR TITLE
IGVF-6-inserts

### PR DESCRIPTION
**Pull request by ian-whaling**
Added users, labs, awards from the DRAFT IGVF Metadata

User emails are not actual emails, just their first name + email domain

### CODE REVIEW

File: src/igvfd/tests/data/inserts/user.json [hunk 1]
```diff
@@ -308,13 +308,1417 @@
...
+        "status": "current",
+        "uuid": "3a0ff5da-c893-4e36-9d0e-d97940a7e13d"
+    },
+    {
+        "email": "buenrostro@email_domain.com",
+        "first_name": "Jason",
+        "last_name": "Buenrostro",
+        "lab": "/labs/jason-buenrostro/",
+        "viewing_groups": [
+            "community"
```
Comment (cricketsloan) [inline src/igvfd/tests/data/inserts/user.json:1280]: Shouldn't the default viewing group for the majority of these people be IGVF not community?

**Review by cricketsloan — CHANGES_REQUESTED** [review]
Ian, thanks for putting this together, it will close out a bunch of related tickets as well.  I have a few questions in line.  What I did not validate was the content, specifically that all of the people and labs are connected properly.

File: src/igvfd/tests/data/inserts/award.json [hunk 2]
```diff
@@ -13,5 +13,173 @@
         "title": "A Data and Administrative Coordinating Center for the Impact of Genomic Variation on Function Consortium",
         "url": "https://www.genome.gov/Funded-Programs-Projects/Impact-of-Genomic-Variation-on-Function-Consortium",
         "viewing_group": "IGVF"
+    },
+    {
+        "name": "1UM1HG011972-01",
+        "project": "IGVF",
+        "title": "Stanford Center for Connecting DNA Variants to Function and Phenotype",
+        "viewing_group": "IGVF",
+        "status": "current"
```
[the remark this answers is not in the document]
Comment (ian-whaling) [reply src/igvfd/tests/data/inserts/award.json]: start dates aren't listed on the spreadsheet unless I should use a generic start_date (2021-09-01)? It is apparently in question if we even need start date. I added everything else

[the remark this answers is not in the document]
Comment (cricketsloan) [reply src/igvfd/tests/data/inserts/award.json]: Is the idea to only have end date?  If we continue past the original awards we will need some sort of date to distinguish them.  But I am content leaving this off for now.

**Pull request merged**